v201603/ad_group_criterion.go

```diff
@@ -164,6 +164,7 @@ func (s AdGroupCriterionService) Get(selector Selector) (adGroupCriterions AdGro
 	}{}
 	err = xml.Unmarshal([]byte(respBody), &getResp)
 	if err != nil {
+		fmt.Println(err)
```

Comment:
@zak10 can you remove this? Looks like it was left over from debugging and I don't think we need this since we are bubbling the error up.